### PR TITLE
Update setup instructions for MacOS

### PR DESCRIPTION
**Pull request by jaucourt**
https://eaflood.atlassian.net/browse/PSD-283

Added developer device setup instructions for a MacOS device

### CODE REVIEW

File: docs/developer-laptop-setup/setup-macbook.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Setup developer macbook
+Written for use with bash, zsh instructions will be similar
+
+1. Install developer tools: from a bash prompt, do `xcode-select --install` (more details here: http://osxdaily.com/2014/02/12/install-command-line-tools-mac-os-x/)
+2. Install nvm: from bash prompt, do
+```
+$ touch ~/.bash_profile
```
Comment (neilbmclaughlin) [inline docs/developer-laptop-setup/setup-macbook.md]: The convention in the other setup guides is to exclude the prompt.

File: docs/developer-laptop-setup/setup-macbook.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+$ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
+$ . ~/.bash_profile
+```
+(see https://github.com/nvm-sh/nvm#installation-and-update for full instructions, including zsh variant)
+
+3. Install node 10:
+```
+$ nvm install 10.0
+$ nvm user 10.0
```
Comment (neilbmclaughlin) [inline docs/developer-laptop-setup/setup-macbook.md]: This should be `nvm use 10.0`

**Review by neilbmclaughlin — CHANGES_REQUESTED** [review]
Just a couple of small changes

File: docs/developer-laptop-setup/setup-macbook.md (hunk 1 above)
Comment (jaucourt) [reply docs/developer-laptop-setup/setup-macbook.md]: Ah, old habits die hard!

File: docs/developer-laptop-setup/setup-macbook.md (hunk 2 above)
Comment (jaucourt) [reply docs/developer-laptop-setup/setup-macbook.md]: Typo, good spot!

**Review by neilbmclaughlin — APPROVED** [review]

**Pull request merged**